numba/tests/npyufunc/test_gufunc.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import, print_function, division
+from numba import unittest_support as unittest
+from numba.decorators import jit
+from numba import float32
+import numpy as np
+import numpy.core.umath_tests as ut
+from numba.npyufunc import GUVectorize
+
+
+def matmulcore(A, B, C):
+    m, n = A.shape
+    n, p = B.shape
+    for i in range(m):
+        for j in range(p):
+            C[i, j] = 0
+            for k in range(n):
+                C[i, j] += A[i, k] * B[k, j]
+
+
+class TestGUFunc(unittest.TestCase):
```

Comment:
We already have guvectorize tests in test_ufuncbuilding and test_guvectorize_scalar. Is there anything that justifies a separate test file for these tests?